hudi-platform-service/hudi-metaserver/hudi-metaserver-client/src/main/java/org/apache/hudi/common/table/HoodieTableMetaserverClient.java

```diff
@@ -60,20 +62,22 @@ public HoodieTableMetaserverClient(Configuration conf, String basePath, Consiste
                                      String databaseName, String tableName, HoodieMetaserverConfig config) {
```

Comment:
I'm not saying to change the member variable as Option, I'm talking about the params in the constructor.